bot/code_review_bot/tasks/clang_tidy.py

```diff
@@ -78,6 +85,29 @@ def __init__(
         self.reliability = reliability
         self.publishable_check = publish
         self.reason = reason
+        self.diag_type = diag_type
```

Comment:
We could just use the already existing `level` attribute, why did you introduce this new attribute?

Reply:
I wanted to better state that we are dealing there with a `diagnostic message`, but if you think there is no need to be extra explicit I'm fine removing it.

Reply:
Yeah I'd say better to simplify